plugins/module_utils/redfish_utils.py

```diff
@@ -298,6 +308,59 @@ def delete_request(self, uri, pyld=None):
                     'msg': "Failed DELETE request to '%s': '%s'" % (uri, to_text(e))}
         return {'ret': True, 'resp': resp}
 
+    @staticmethod
+    def _prepare_multipart(fields):
```

Comment:
@felixfontein is doing something like this okay? I saw something similar done in plugins/modules/jira.py. Another alternative would be to import requests_toolbelt, but that seemed like it would be frowned upon.

Reply:
This should be OK. It might make sense to move this into a generic module utils that could be used by other kind of modules (and module utils) as well.